diff --git a/stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp b/stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp
index e0ceb6a19..e3c8fd890 100644
--- a/stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp
+++ b/stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp
@@ -60,14 +60,13 @@ namespace ams::mitm::settings {
 
             /* Modify the atmosphere firmware version to display a custom version string. */
             {
-                const auto api_info = exosphere::GetApiInfo();
                 const char emummc_char = emummc::IsActive() ? 'E' : 'S';
 
                 /* NOTE: We have carefully accounted for the size of the string we print. */
                 /* No truncation occurs assuming two-digits for all version number components. */
                 char display_version[sizeof(g_ams_firmware_version.display_version)];
 
-                util::SNPrintf(display_version, sizeof(display_version), "%s|AMS %u.%u.%u|%c", g_ams_firmware_version.display_version, api_info.GetMajorVersion(), api_info.GetMinorVersion(), api_info.GetMicroVersion(), emummc_char);
+                util::SNPrintf(display_version, sizeof(display_version), "%s EVARON-AWS 0.2.0  %c", g_ams_firmware_version.display_version, emummc_char);
 
                 std::memcpy(g_ams_firmware_version.display_version, display_version, sizeof(display_version));
             }
